From 18025e1e29b228d9f3390a2822ce9b2de891806c Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Fri, 21 Sep 2018 15:53:53 +0530
Subject: [PATCH 0427/1852] documentation: dt: vpss-scaler: Add support for max
 width and height

commit 4ff5aa9f4bd0357a121f88ff2ba442e5192f2091 from
https://github.com/Xilinx/linux-xlnx.git

Adds two optional parameters xlnx,max-width and xlnx,max-height which
determine the max size of frame the IP can process.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,v-vpss-scaler.txt  | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-scaler.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-scaler.txt
index 0fbbf182b328..cc0616784bbc 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-scaler.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,v-vpss-scaler.txt
@@ -36,6 +36,11 @@ Required port properties:
 
 - xlnx,video-width: Video width as defined in video.txt
 
+Optional properties:
+
+- xlnx,max-height: Maximum number of lines.
+- xlnx,max-width: Maximum number of pixels in a line.
+
 Example:
 
 	scaler_1:scaler@a0000000 {
-- 
2.31.1

